code/game/gamemodes/miniantags/borer/borer_chemicals.dm

```diff
@@ -1,59 +1,82 @@
 /datum/borer_chem
 	var/chemname
 	var/chem_desc = "This is a chemical"
+	var/chem_message //Text sent to the host when injecting chemicals
 	var/chemuse = 30
 	var/quantity = 10
 
 /datum/borer_chem/epinephrine
 	chemname = "epinephrine"
 	chem_desc = "Stabilizes critical condition and slowly restores oxygen damage. If overdosed, it will deal toxin and oxyloss damage."
+	chem_message = "<span class='notice'>You feel a surge of epinephrine flooding your bloodstream!</span>"
```

Comment:
I gotta agree that it felt a bit... weird. But I'm having trouble describing epinephrine without making it sound like meth or something like that. I thought of writing something about stablizing or maybe about the oxygen effect, but truth be told, epinephrine does almost nothing if you're not in crit...
I guess IRL you'd feel a rush of adrenaline, but adrenaline is just another name for it :S.